### DISCUSSION TITLE
Turn Off Maintenance mode v 6.05

### DISCUSSION THREAD

**Discussion by mortensenk-osp** [pimcore/pimcore#8868] [Q&A / Support] [answered]
I turned on Maintenance mode thinking I could disable it from the console. I can't find out how to turn it off.

I have only seen solutions for later versions.
Any help is greatly appreciated.

**Comment by NiklasBr** [comment, marked as answer]
The command should be `bin/console pimcore:maintenance-mode --disable`

It is also possible to list all available commands by just running the command `bin/console` and then `grep` the result if you want.

**Reply by mortensenk-osp** [reply]
Thanks for your response. That command isn't available in my version (6.05).

**Reply by NiklasBr** [reply]
Oh right, sorry!

There should be a way to disable it by removing the file placed at `PIMCORE_CONFIGURATION_DIRECTORY . '/maintenance.php'` or by running [public static function deactivateMaintenanceMode()](https://github.com/pimcore/pimcore/blob/v6.0.5/lib/Tool/Admin.php#L148-L180)

**Reply by mortensenk-osp** [reply]
Thanks. I will try the function. That file doesn't exist. I redeployed and it came up fine.

**Comment by ShawnCrigger-SOLS** [comment]
None of these options seem to disable maintenance tasks. I may be thinking the wrong thing.  Is there a way to disable, say, gzipping the log directory? Or any of the other maintenance tasks that is listed when I run ` bin/console help maintenance`
I get this huge list.


 - any bundle class name handling maintenance (e.g. PimcoreEcommerceFrameworkBundle)
 - scheduledtasks
 - logmaintenance
 - cleanuplogfiles
 - httperrorlog
 - archiveLogEntries
 - runSanityCheck
 - cleanupBrickTables
 - cleanupClassificationstoreTables
 - cleanupFieldcollectionTables
 - versioncleanup
 - versioncompress
 - VersionsCleanupStackTraceDb
 - redirectcleanup
 - cleanupbrokenviews
 - cleanupcache
 - tmpstorecleanup
 - imageoptimize
 - asset_document_convert
 - housekeeping
 - low_quality_image_preview
 - markexpiredtagsdisabled

How can I disable some of these tasks?
